addons/project_timesheet_holidays/models/account_analytic.py

```diff
@@ -11,7 +11,28 @@ class AccountAnalyticLine(models.Model):
     holiday_id = fields.Many2one("hr.leave", string='Leave Request')
     global_leave_id = fields.Many2one("resource.calendar.leaves", string="Global Time Off", ondelete='cascade')
 
+    def _get_redirect_action(self):
+        action_data = {
+           'name': _('Time Off'),
+           'type': 'ir.actions.act_window',
+           'res_model': 'hr.leave',
+        }
+        if len(self.mapped("holiday_id")) == 1:
+            action_data.update({
+                'res_id': self.mapped("holiday_id")[0].id,
+                'views': [(self.env.ref('hr_holidays.hr_leave_view_form').id, 'form')],
```

Comment:
Perhaps `self.env.ref('hr_holidays.hr_leave_view_form').id` is worth a variable?